Gems/ROS2/Code/Include/ROS2/ROS2GemUtilities.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "AzCore/Component/ComponentBus.h"
+#include "AzCore/Component/Entity.h"
+#ifdef ROS2_EDITOR
+#include "AzToolsFramework/ToolsComponents/GenericComponentWrapper.h"
+#endif
+namespace ROS2
+{
+    namespace Utils
+    {
+        /// Create component for a given entity in safe way.
+        /// \param entityId entity that will own component
+        /// \param componentType Uuid of component to create
+        /// \return created componentId, if it fails, it returns invalid id
+        AZ::ComponentId CreateComponent(const AZ::EntityId entityId, const AZ::Uuid componentType);
+
+        /// Retrieve component from entity given by a pointer. It is a way to get game components and wrapped components.
+        /// We should use that that we are not sure if we access e.g. ROS2FrameComponent in game mode or from Editor
+        /// \param entity pointer to entity e.g. with GetEntity()
+        /// \return pointer to component with type T
```

Comment:
Should be ```@param``` and ```@return``` instead
```suggestion
        /// @param entity pointer to entity e.g. with GetEntity()
        /// @return pointer to component with type T
```